table.go

```diff
@@ -17,6 +17,9 @@ type Table struct {
 
 // Column defines a column with name and type.
 func (t *Table) Column(name string, typ ColumnType, options ...ColumnOption) {
+	if typ == BigID || typ == ID {
+		options = append([]ColumnOption{Primary(true)}, options...)
```

Comment:
why not this?
```suggestion
		options = append(options, Primary(true))
```

Reply:
Because than you can not override it as default `Primary(true)` will be executed last and will override if you have specified `Primary(false)` in `options`

Reply:
make sense